modules/monitoring-default-monitoring-components.adoc

```diff
@@ -23,14 +23,8 @@ By default, the {product-title} {product-version} monitoring stack includes thes
 |Prometheus
 |Prometheus is the monitoring system on which the {product-title} monitoring stack is based. Prometheus is a time-series database and a rule evaluation engine for metrics. Prometheus sends alerts to Alertmanager for processing.
 
-|Prometheus Adapter
-|The Prometheus Adapter (PA in the preceding diagram) translates Kubernetes node and pod queries for use in Prometheus. The resource metrics that are translated include CPU and memory utilization metrics. The Prometheus Adapter exposes the cluster resource metrics API for horizontal pod autoscaling. The Prometheus Adapter is also used by the `oc adm top nodes` and `oc adm top pods` commands.
-
-|Metrics Server (Technology Preview)
-|If enabled, the Metrics Server component collects resource metrics and exposes them in the `metrics.k8s.io` Metrics API service for use by other tools and APIs, which frees the core platform Prometheus stack from handling this functionality.
-
-The Technology Preview of the Metrics Server component is automatically installed instead of Prometheus Adapter if you configure the `FeatureGate` custom resource with the `TechPreviewNoUpgrade` option.
-For more information about the support scope of Red{nbsp}Hat Technology Preview features, see link:https://access.redhat.com/support/offerings/techpreview/[Technology Preview Features Support Scope].
+|Metrics Server
+|The Metrics Server component collects resource metrics and exposes them in the `metrics.k8s.io` Metrics API service for use by other tools and APIs, which frees the core platform Prometheus stack from handling this functionality. Note that with the {product-title} 4.16 release, Metrics Server replaces Prometheus Adapter.
```

Comment:
```suggestion
|The Metrics Server component (MS) collects resource metrics and exposes them in the `metrics.k8s.io` Metrics API service for use by other tools and APIs, which frees the core platform Prometheus stack from handling this functionality. Note that with the {product-title} 4.16 release, Metrics Server replaces Prometheus Adapter.
```
What do you think about the suggestion, add `MS` so user would know `MS` stands for Metrics Server in the structure graph

Reply:
Good idea! Added.